_releaser/_website-config-docs-lab.json

```diff
@@ -0,0 +1,388 @@
+{
+  "ErrorDocument": {
+    "Key": "404.html"
+  },
+  "IndexDocument": {
+    "Suffix": "index.html"
+  },
+  "RedirectAllRequestsTo": null,
+  "RoutingRules": [
+    {
+      "Condition": {
+        "HttpErrorCodeReturnedEquals": null,
+        "KeyPrefixEquals": "v1.4/"
+      },
```

Comment:
Do you think we should consider to "skip" these redirects for the labs domain? (I'm fine either way; mostly thinking; we're probably won't be using these on that domain - or are we testing against them?)

Reply:
Yes in a follow-up but for now we keep them as @VictorBersy will first create the lab environment so we can test everything is ok with the current workflow and after that we take into account https://github.com/docker/docker.github.io/pull/15151